.travis.yml

```diff
@@ -63,6 +63,9 @@ install:
         curl -L https://github.com/google/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip -o /tmp/protoc.zip
         unzip /tmp/protoc.zip -d "$HOME"/protoc
     fi
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install 8.16.0
+  - npm install
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
why do we need this unconditionally in all matrix runs? It's only needed for steps where the UI is built, and I think it's already being installed there